proto/autokitteh/remote/v1/runner_manager_svc.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package autokitteh.remote.v1;
+
+import "autokitteh/remote/v1/types.proto";
+
+message ContainerConfig {
+  string image = 1;
+  // TBD by @efiShtain
+}
+
+// TODO: Will become Start once we split to files
+// Tell runner manager to start a runner
+message StartRunnerRequest {
+  ContainerConfig container_config = 1;
+
+  // user code as tar archive
+  bytes build_artifact = 2;
+  // vars from manifest, secrets and connections
+  map<string, string> vars = 3;
+
+  string worker_address = 4;
+}
+
+message StartRunnerResponse {
+  string runner_id = 1;
+  string runner_address = 2;
+  string error = 3;
+}
+
+message RunnerHealthRequest {
+  string runner_id = 1;
+}
+
+message RunnerHealthResponse {
+  bool healthy = 1;
+  string error = 2;
+}
+
+message StopRunnerRequest {
+  string runner_id = 1;
+}
+
+message StopRunnerResponse {
+  string error = 1;
+}
+
+message RunnerManagerHealthRequest {}
```

Comment:
All health requests and responses (except RunnerHealth) are the same, use the same message?